CODE_OF_CONDUCT.md

```diff
@@ -14,7 +14,7 @@ diverse, inclusive, and healthy community.
 
 ## Our Standards
 
-Examples of behavior that contributes to a positive environment for our
+Examples of behaviour that contributes to a positive environment for our
```

Comment:
This isn't a spelling mistake. It's US English and you're following UK English. Both are correct and can be used in the preferred language style.